parser/parser_test.go

```diff
@@ -581,17 +581,6 @@ func TestFromFile(t *testing.T) {
 				assert.Equal(t, "SendTextForHighPriority", w.States[10].SwitchState.DefaultCondition.Transition.NextState)
 				assert.Equal(t, true, w.States[10].End.Terminate)
 			},
-		}, {
-			"./testdata/workflows/dataInputSchemaValidation.yaml", func(t *testing.T, w *model.Workflow) {
-				assert.NotNil(t, w.DataInputSchema)
```

Comment:
isn't it better to keep it?

Reply:
No this validation is not valid anymore since Schema is not loading the external file to the attribute.